prusa_slicer_post_processing_script.py

```diff
@@ -277,15 +277,18 @@ def main(gCodeFileStream,path2GCode,skipInput)->None:
                         #plt.axis('square')
                         #plt.show()
                         for ida,arc in enumerate(arcs4gcode):
-                            if not arc.is_empty:    
+                            if not arc.is_empty:
                                 arcGCode=arc2GCode(arcline=arc,eStepsPerMM=eStepsPerMM,arcidx=ida,kwargs=parameters)
                                 arcOverhangGCode.append(arcGCode)
                                 if parameters.get("TimeLapseEveryNArcs")>0:
                                     if ida%parameters.get("TimeLapseEveryNArcs"):
                                         arcOverhangGCode.append("M240\n")
 
-                #apply special cooling settings:    
-                if len(layer.oldpolys)>0:
+                        modify=True
+                        gcodeWasModified=True
```

Comment:
We moved these to the bottom of the loop so that they're only made true if arc generation was a success.